hapi/hedera-protobufs/services/state/roster/roster.proto

```diff
@@ -45,10 +45,16 @@ message Roster {
  *
  * Each roster entry SHALL encapsulate the elements required
  * to manage node participation in the Threshold Signature Scheme (TSS).<br/>
- * All fields except tss_encryption_key are REQUIRED.
+ * All fields are REQUIRED.
  */
 message RosterEntry {
 
+    // The tssEncryptionKey field has been reserved because it is no longer
+    // required to be stored in the Roster. The public portion of this key will
+    // continue to be stored in the Roster state, but will, however, be stored
```

Comment:
The public portion of the `tssEncryptionKey` will need to be stored in `TssBaseService`, not the `Roster state`